src/TouristInfoCards/TouristInfoCards.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import TouristInfoCard from "./TouristInfoCard/TouristInfoCard";
+
+import Cities from "../data/Cities";
```

Comment:
In React, we usually only start variable names with a capital letter when they are components